pkgs/servers/pulseaudio/default.nix

```diff
@@ -59,7 +59,7 @@ stdenv.mkDerivation rec {
     ./0007-Fix-link-args-on-darwin.patch
   ];
 
-  outputs = [ "out" "dev" ];
+  outputs = [ "out" "dev" "config-header" ];
```

Comment:
Why not just `header`?

Reply:
BTW, is there any standard output for those files?

Reply:
> Why not just `header`?

What header? The name should be descriptive. 
Normal headers are in dev

Reply:
Maybe it could be pulseaudio-config.h in dev since the file will have to be copied anyway to be used.

Reply:
I think this approach makes more sense.

Reply:
Can you explain why it should be a new output and not in dev like normal headers?

Reply:
It's required for one specific package (a pulseaudio module for xrdp)

Reply:
It doesn't even build now
```
/nix/store/jivxp510zxakaaic7qkrb7v1dd2rdbw9-multiple-outputs.sh: line 113: config-header: invalid variable name
```

To me somewhere in the .dev output still sounds better.  I can't see why not.  And in my eyes, adding a small file (5KB here) into an existing output is smaller overhead than creating a new output; .dev shouldn't even get retained in runtime closures.

Reply:
ok then